packages/web-components/src/rating-display/rating-display.ts

```diff
@@ -117,6 +63,24 @@ export class RatingDisplay extends FASTElement {
     return this.count ? this.intlNumberFormatter.format(this.count) : '';
   }
 
+  /**
+   * Gets the selected value
+   *
+   * @internal
+   */
+  public getSelectedValue(): number {
```

Comment:
Thanks for adding these! One small thing, looks like they can be marked as `protected` instead of `public` because they're only used in this and the extended class.